packages/framework/tests/Feature/Services/Markdown/DynamicMarkdownLinkProcessorTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/** @noinspection HtmlUnknownTarget */
+
+declare(strict_types=1);
+
+namespace Hyde\Framework\Testing\Feature\Services\Markdown;
+
+use Hyde\Pages\InMemoryPage;
+use Hyde\Testing\UnitTestCase;
+use Hyde\Support\Models\Route;
+use Hyde\Support\Facades\Render;
+use Hyde\Foundation\Facades\Routes;
+use Hyde\Support\Models\RenderData;
+use Hyde\Markdown\Processing\DynamicMarkdownLinkProcessor;
+
+/**
+ * @covers \Hyde\Markdown\Processing\DynamicMarkdownLinkProcessor
+ *
+ * @see \Hyde\Framework\Testing\Feature\DynamicMarkdownLinksFeatureTest
+ */
+class DynamicMarkdownLinkProcessorTest extends UnitTestCase
```

Comment:
Needs tests for when route is missing (throw?), and from nested pages (main benefit of this)